Do not omit bubble for subsequent comments with delay between them

davidmz opened the pull request:
Bubbles omits only if interval between comments by same author is less than 10 minutes. It closes T58.

dsumin commented on src/components/select-utils.js:
```diff
@@ -63,8 +63,11 @@ export const joinPostData = state => postId => {
+    const omitBubble = omitRepeatedBubbles 
+      && postViewState.omittedComments === 0 
+      && comment.createdBy === previousPost.createdBy 
+      && comment.createdAt - previousPost.createdAt < 600000; // max 10 min between comments
```
@davidmz давай вынесем это в константу в начало модуля?

davidmz replied:
да, можно.

davidmz committed:
Do not omit bubble for subsequent comments with delay between them

Bubbles omits only if interval between comments by same author is less than 10 minutes. It closes T58.
